public/app/plugins/datasource/postgres/module.ts

```diff
@@ -8,8 +8,9 @@ class PostgresConfigCtrl {
 
   /** @ngInject **/
   constructor($scope) {
-    this.current.jsonData.sslmode = this.current.jsonData.sslmode || 'require';
+    this.current.jsonData.sslmode = this.current.jsonData.sslmode || 'verify-full';
```

Comment:
A bit unsure why this change is needed or if it was mistakenly added?

Reply:
This is unrelated i'll do a separate pr for this.